scripts/ci/update-translations.sh

```diff
@@ -3,7 +3,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 source ${DIR}/../version_number.sh
 
-lupdate -recursive ${DIR} -ts ${DIR}/i18n/qfield_en.ts
+lupdate -recursive ${CI_BRANCH} -ts ${CI_BRANCH}/i18n/qfield_en.ts
```

Comment:
What's this change here for?

Reply:
Otherwise it's looking for the files in `scripts/ci/`, no?
Could be `${DIR}/../..` as well...

Reply:
`CI_BRANCH` is wrong of course...